elections-common/src/main/resources/config.yml

```diff
@@ -40,3 +40,6 @@ rank:
   command: "none"
   # Who should win if there are two or more winners?
   strategy: "none" # None, both
+
+update:
+  check: false
```

Comment:
Please default this to true

Reply:
will do

Reply:
fixed in 84eb15f68451a5556cc9098822d5f2a83c28e16b